contrib/abseil-cpp-cmake/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(ABSL_ROOT_DIR "${ClickHouse_SOURCE_DIR}/contrib/abseil-cpp")
-set(BUILD_TESTING OFF)
```

Comment:
is it removed or?

Reply:
It was and it is off by default in abseil build description (called in l. 4). No need to disable it explicitly.